packages/ui/src/components/ChatMessage/ChatMessage.tsx

```diff
@@ -186,6 +215,15 @@ const ChatMessage = ({
       post.replyContactIds ? (
         <ChatMessageReplySummary post={post} onPress={handleRepliesPressed} />
       ) : null}
+      <ActionSheet open={showRetrySheet} onOpenChange={setShowRetrySheet}>
```

Comment:
interesting that we're including a sheet with every post, did you notice any performance impact? Wondering if it might be worth pulling up and handling as a singleton.

May be fine as is, just know we've run into issues with rendering actions per item in a list before on web

Reply:
Haven't noticed any performance impact on android or iOS, but I'll run the profiler and see if anything pops out.

Reply:
Feel free to merge if it seems fine

Reply:
After opening channels, threads, and scrolling around, not seeing any perf issues in the profiler.